backends/cuda/ceed-cuda-qfunction.c

```diff
@@ -164,26 +177,34 @@ int CeedQFunctionCreate_Cuda(CeedQFunction qf) {
   CeedQFunction_Cuda *data;
   ierr = CeedCalloc(1, &data); CeedChk(ierr);
   ierr = CeedQFunctionSetData(qf, data); CeedChk(ierr);
-  CeedInt numinputfields, numoutputfields;
-  ierr = CeedQFunctionGetNumArgs(qf, &numinputfields, &numoutputfields);
-  CeedChk(ierr);
```

Comment:
Did these lines serve a purpose?

Reply:
I think those predate single source Qfunctions and were missed in the transition